core/src/main/java/fr/sncf/osrd/railjson/parser/RJSRollingStockParser.java

```diff
@@ -92,15 +106,49 @@ private static RJSRollingResistance.Davis parseRollingResistance(
         return (RJSRollingResistance.Davis) rjsRollingResistance;
     }
 
-    // these are commented as they'll be used soon for profiles
+    /** Parse an RJSEffortCurveConditions into a EffortCurveConditions */
+    private static RollingStock.EffortCurveConditions parseEffortCurveConditions(
+            RJSEffortCurves.RJSEffortCurveConditions rjsCond,
+            String fieldKey
+    ) {
+        if (rjsCond == null)
+            throw new MissingRollingStockField(fieldKey);
+        return new RollingStock.EffortCurveConditions(parseComfort(rjsCond.comfort));
+    }
+
+    /** Parse rjsComfort into a RollingStock comfort */
+    public static RollingStock.Comfort parseComfort(RJSComfortType rjsComfort) {
+        if (rjsComfort == RJSComfortType.AC)
+            return RollingStock.Comfort.AC;
+        if (rjsComfort == RJSComfortType.HEATING)
+            return RollingStock.Comfort.HEATING;
+        return RollingStock.Comfort.STANDARD;
```

Comment:
```suggestion
            return RollingStock.Comfort.HEATING;
        else 
            return RollingStock.Comfort.STANDARD;
```

Reply:
It is not necessary to put an `else` here.